src/analysis/interproc/interproc.rs

```diff
@@ -66,15 +65,11 @@ impl<'a,'b, M, T> InterProcAnalyzer<'a, 'b, M, T>
 #[cfg(test)]
 mod test {
     use super::*;
-    use frontend::source::{FileSource, Source};
```

Comment:
Well, it still doesn't use `frontend::source::Source`, you mean usage of `r2pipe::r2::R2`?